scripts/repositories.json

```diff
@@ -4,7 +4,7 @@
             "name": "obs-studio-node",
             "url": "https://s3-us-west-2.amazonaws.com/obsstudionodes3.streamlabs.com/",
             "archive": "osn-[VERSION]-release-[OS].tar.gz",
-            "version": "0.18.1",
+            "version": "0.16.21-capture6",
```

Comment:
What does this change mean?

Reply:
That's temporary.  Will remove this change before merging once OSN is updated.